kong/plugins/moesif/log.lua

```diff
@@ -132,6 +147,21 @@ function get_config_internal(conf)
 end
 
 
+-- Get App Config function for Governance Rules
+-- @param `conf`     Configuration table, holds http endpoint details
+function _M.get_config_for_rules(conf)
+  local ok, err = pcall(get_config_internal, conf)
```

Comment:
hmm.. you are using same fcn get_config_internal for get rules config and also get config? am I missing something?

Reply:
We're using it to get the app config only because we need to know the response Tag to determine when to fetch the update governance rules here - https://github.com/Moesif/kong-plugin-moesif/pull/33/files#diff-bf25e21931600691cf250b96dcae5bfdR241